apps/qa/src/template/template.py

```diff
@@ -0,0 +1,110 @@
+import streamlit as st
+from src.components import sidebar
+from src.components.build_outputs import (
+    load_build_outputs,
+    generate_geo_data,
+    generate_maps,
+    show_build_output,
+    show_map,
+)
+
+
+def template():
+    st.title("QA of Template DB")
+    product_key = sidebar.data_selection("db-template")
+
+    csv_files = [
```

Comment:
It would be nice to use `product_key` to grab data from S3

Reply:
I think it is, further down on [line 57](https://github.com/NYCPlanning/data-engineering/blob/dm-new-qa-pages/apps/qa/src/template/template.py#L57)

the `csv_files` list here doesn't feel ideal but I imagined we won't always QA all output files to the same degree so tried this approach to declare the sort of "primary" csv files to QA